combat-trainer.lic

```diff
@@ -650,6 +654,7 @@ class SpellProcess
       @offensive_spells
         .select { |spell| spell['expire'] }
         .reject { |spell| spell['cyclic'] }
+        .reject { |spell| !spell['expire_on_death'] }
```

Comment:
I think maybe the variable can be changed to something more meaningful... `expires_in_time` or maybe `heavy_spell` (though that might be too constraining)